src/main/java/com/example/waggle/global/util/ObjectUtil.java

```diff
@@ -30,4 +35,37 @@ public static void validateKeywordLength(String keyword) {
     public static <T> boolean doesNotContain(List<T> objectList, T target) {
         return !objectList.contains(target);
     }
+
+    public static String serialize(Object object) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        String result = null;
+        try {
+            result = objectMapper.writeValueAsString(object);
+        } catch (JsonProcessingException e) {
+            throw new RuntimeException(e);
+        }
+        return result;
+    }
```

Comment:
Exception이 터지게 되면 아래 return result; 구문이 실행되지 않기 때문에, try 문 내에서 직접 return을 해줘도 될 것 같습니다! (다른 메서드들도 마찬가지)
그리고 ObjectMapper 객체도 final static 변수로 선언해주면 좋을 것 같습니당

```suggestion
    private static final OBJECT_MAPPER = new ObjectMapper();

    public static String serialize(Object object) {
            try {
                return OBJECT_MAPPER.writeValueAsString(object);
            } catch (JsonProcessingException e) {
                throw new RuntimeException("Failed to serialize object", e);
            }
    }
```

Reply:
exception이 발생하면 result를 리턴받지 못하는 것은 똑같지 않나요?